hoomd/util.py

```diff
@@ -10,6 +10,7 @@
 from inspect import isclass
 from copy import deepcopy
 from hoomd.trigger import PeriodicTrigger
+from hoomd.variant import Constant
 
 ## \internal
 # \brief Compatibility definition of a basestring for python 2/3
```

Comment:
This Python 2 compatibility layer can be cut out (in another PR? you can decide).